telegram/ext/_handlers/chatmemberhandler.py

```diff
@@ -49,10 +51,16 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+
         chat_member_types (:obj:`int`, optional): Pass one of :attr:`MY_CHAT_MEMBER`,
             :attr:`CHAT_MEMBER` or :attr:`ANY_CHAT_MEMBER` to specify if this handler should handle
             only updates with :attr:`telegram.Update.my_chat_member`,
             :attr:`telegram.Update.chat_member` or both. Defaults to :attr:`MY_CHAT_MEMBER`.
+
+            .. versionadded:: NEXT.VERSION
```

Comment:
this should go *below* the description of the `chat_id` parameter :)

please also remove the added empty newlines